Library/ViewModels/CommentCellViewModelTests.swift

```diff
@@ -79,75 +85,63 @@ internal final class CommentCellViewModelTests: TestCase {
 
       self.bottomRowStackViewIsHidden
         .assertValue(
-          false,
-          "The feature flag is false and the user is a backer, therefore the stack view is not hidden."
+          true,
+          "The feature flags are false, therefore the stack view is hidden."
         )
     }
   }
 
   func testOutputs_bottomRowStackViewIsHidden_LoggedOut_FeatureFlag_True() {
```

Comment:
`isNotHidden`

Reply:
`bottomRowStackViewIsHidden` is the name of the output I'm testing so I'm leaving the name as is so we can be aware of which output the test is focusing on.

Reply:
Resolving the ones below.